lib/request_builder.rb

```diff
@@ -19,6 +20,8 @@ def initialize(bag_id:, content_type:, external_id:, user:)
   end
 
   def create
+    Rails.logger.debug "making directory #{request.upload_path}"
+    fs.mkdir_p request.upload_path
```

Comment:
Unnecessary - the expectation is that the client has permissions to create this path